.github/workflows/releaser.yaml

```diff
@@ -0,0 +1,45 @@
+name: Releaser
+
+on:
+  workflow_dispatch:
+    inputs:
+      base_branch:
+        description: 'The base branch to release from'
+        required: true 
+      sync_operator_repository:
+        description: "Sync operator repository"
+        required: false
+        default: "noobaa-operator"
+
+permissions:
+  contents: write
+
+jobs:
+  releaser:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set branch
+        run: echo "BRANCH=${{ github.event.inputs.base_branch }}" >> $GITHUB_ENV
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ env.BRANCH }}
+      - name: Fetch all tags
+        run: git fetch --force --tags
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16.14.0 
```

Comment:
Change to node 18.12.1
Can we take it out of the .nvmrc?

Reply:
Using node-version-file now which reads the version from nvmrc file. Using https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#node-version-file.